FairBid/tapjoy.plist.xml

```diff
@@ -28,6 +28,290 @@
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>cstr6suwn9.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>yclnxrl5pm.skadnetwork</string>
+			</dict>
+			<dict>
+				<key>SKAdNetworkIdentifier</key>
+				<string>xy9t38ct57.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>nzq8sh4pbs.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>3qy4746246.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>tl55sbb4fm.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>y45688jllp.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>7rz58n8ntl.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>52fl2v3hgk.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>yrqqpx2mcb.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>97r2b46745.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>gvmwg8q7h5.skadnetwork</string>
+			</dict>
+			<dict>
+				<key>SKAdNetworkIdentifier</key>
+				<string>w9q455wk68.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>lr83yxwka7.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>dzg6xy7pwj.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>glqzh8vgby.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>c6k4g5qg8m.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>wg4vff78zm.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>n9x2a789qt.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>4fzdc2evr5.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>klf5c3l5u5.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>ppxm28t8ap.skadnetwork</string>
+			</dict>
+			<dict>
+				<key>SKAdNetworkIdentifier</key>
+				<string>737z793b9f.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>ejvt5qm6ak.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>cg4yq2srnc.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>.skadnetwork</string>
```

Comment:
Pretty sure this is incorrect :)

Reply:
Thank you for pointing it out @thomasmso :)